pkgs/by-name/ce/celeste64/package.nix

```diff
@@ -15,7 +16,7 @@
   xorg,
   libxkbcommon,
   libdrm,
-  withSELinux ? false,
+  withSELinux ? config.selinuxSupport,
```

Comment:
I was quite surprised to see that Celeste 64 has SELinux support, but after looking at its source tree, [I don't think it actually does](https://github.com/search?q=repo%3AEXOK%2FCeleste64%20selinux&type=code)? I imagine what could have happened is that this was copied this from another derivation without realizing that SELinux doesn't actually do anything without support on the application side.

Or, maybe .NET has support for it? Or maybe there's a setup hook I'm missing?

Reply:
Yeah, that comes from #285232 by @itslychee

Reply:
Yeah, I saw. I just didn't think it was worth pinging them; was going to test and drop it myself if it turns out it's extraneous.